static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -433,37 +436,95 @@ const SPECIAL_FIELDS: SpecialFields = {
   },
   assignee: {
     sortField: 'assignee.name',
-    renderFunc: data => {
-      const assignedTo: Actor = {
-        type: data['assignee.type'],
-        id: data['assignee.id'],
-        name: data['assignee.name'],
-        email: data['assignee.email'],
-      };
+    renderFunc: (_, {issueData}) => assigneeRenderer(issueData),
+  },
+};
 
-      return (
-        <ActorContainer>
-          {assignedTo.type && assignedTo.id && assignedTo.name ? (
-            <ActorAvatar
-              actor={assignedTo}
-              size={28}
-              tooltip={t(
+const assigneeRenderer = (issueData?: IssueTableData) => {
+  const {assignedTo, suggestedAssignees} = issueData
+    ? issueData
+    : {assignedTo: undefined, suggestedAssignees: undefined};
+  const suggestedReasons: Record<SuggestedOwnerReason, React.ReactNode> = {
+    suspectCommit: tct('Based on [commit:commit data]', {
```

Comment:
I would probably key this as `commitLink`, to make it more clear

Reply:
Removed this change, it is no longer necessary due to updates to reuse AssigneeSelector!